### PR TITLE
Add support for 8bit images in TotalSegmentator

### LINKED ISSUE #397: Add option to set TotalSegmentator to `uint8` data type
Currently `TotalSegmentator` image types are `int16` - which include negative values as well. It will be great to support the functionality to change the data type to format like `uint8`, which makes it handy to visualise and train on

### PR DESCRIPTION
**Pull request by ioangatop**
Closes #397

### CODE REVIEW

**Review by nkaenzig — COMMENTED** [review]
Looks good! Left some small comments.

File: src/eva/vision/data/datasets/segmentation/total_segmentator.py [hunk 1]
```diff
@@ -127,7 +131,11 @@ def load_image(self, index: int) -> tv_tensors.Image:
         image_path = self._get_image_path(index)
         slice_index = self._get_sample_slice_index(index)
         image_array = io.read_nifti_slice(image_path, slice_index)
-        image_rgb_array = image_array.repeat(3, axis=2)
+        if self._as_uint8:
+            image_array = convert._to_8bit(image_array)
+            image_rgb_array = cv2.cvtColor(image_array, cv2.COLOR_GRAY2RGB)
```
[the remark this answers is not in the document]
Comment (ioangatop) [reply src/eva/vision/data/datasets/segmentation/total_segmentator.py:136]: the cv2 applies a weighted stack of the channels which makes it easier to visualise and maybe more standard - the repeat is sometimes used as in nn maybe it wont matter so much. However we do have to test the repeat in the `int16` case if we get the same/better results

File: src/eva/vision/utils/convert.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""Image conversion related functionalities."""
+
+from typing import Any
+
+import numpy as np
+import numpy.typing as npt
+
+
+def _to_8bit(image_array: npt.NDArray[Any]) -> npt.NDArray[np.uint8]:
```
[the remark this answers is not in the document]
Comment (ioangatop) [reply src/eva/vision/utils/convert.py:9]: not really, I was thinking of using it only inside the datasets other classes, but if we have cases that people can use it their python API, maybe public makes sense

File: src/eva/vision/data/datasets/segmentation/total_segmentator.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ioangatop) [reply src/eva/vision/data/datasets/segmentation/total_segmentator.py]: done!

File: src/eva/vision/utils/convert.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ioangatop) [reply src/eva/vision/utils/convert.py]: done!

[the remark this answers is not in the document]
Comment (ioangatop) [reply src/eva/vision/utils/convert.py]: done!

**Review by nkaenzig — APPROVED** [review]
LGTM 🚀

**Pull request merged**